apps/engine.c

```diff
@@ -258,6 +259,25 @@ static void util_do_cmds(ENGINE *e, STACK_OF(OPENSSL_STRING) *cmds,
     }
 }
 
+struct util_store_cap_data {
+    ENGINE *engine;
+    char **cap_buf;
+    int *cap_size;
```

Comment:
I'm sure, but it's adapted to the second argument for `append_buf`, and that's a different PR, methinks.